core/src/validator.rs

```diff
@@ -835,6 +836,8 @@ fn get_stake_percent_in_gossip(
                 wrong_shred_stake += activated_stake;
                 wrong_shred_nodes.push((*activated_stake, vote_state.node_pubkey));
             }
+        } else if vote_state.node_pubkey == my_id {
+            online_stake += activated_stake; // This node is online
```

Comment:
Restored the `else if`.   The stake of *this* node should certainly considered to be online!

Reply:
Sorry about the extra work! I had a pr to this effect put together first, but convinced myself it was intentional.